wies/projects/templates/assignment_tabs.html

```diff
@@ -7,57 +7,162 @@
   <h1 style="margin: 0;">Opdrachten</h1>
 </div>
 
-<!-- Filter Form -->
-<form id="assignment-tab-filter-form"
-      hx-get="{% url 'assignments' %}"
-      hx-target="#assignment-tabs-section"
-      hx-swap="outerHTML"
-      hx-include="[name='tab']"
-      class="rvo-layout-grid rvo-layout-gap--md rvo-layout-grid-columns--four"
-      style="margin-bottom: 2em;">
-    
-    <!-- Hidden field to maintain active tab -->
-    <input type="hidden" name="tab" value="{{ active_tab }}">
-    
-    <div class="rvo-layout-column rvo-layout-gap--sm">
-        <label for="name-search" class="rvo-label">Zoeken</label>
-        <input type="text" 
-               id="name-search" 
-               name="name" 
-               value="{{ request.GET.name|default:'' }}"
-               placeholder="Zoek op naam, opdrachtgever of ministerie..."
-               class="utrecht-textbox utrecht-textbox--html-input"
-               hx-get="{% url 'assignments' %}" 
-               hx-target="#assignment-tabs-section" 
-               hx-trigger="keyup delay:300ms"
-               hx-include="[name='tab'], [name='order'], [name='skill']">
-    </div>
-    <div class="rvo-layout-column rvo-layout-gap--sm">
-        <label for="skills" class="rvo-label">Rollen:</label>
-        <select name="skill" 
-                id="skills" 
-                class="js-skills-select-single utrecht-select utrecht-select--html-select" 
+<!-- Filters Section -->
```

Comment:
many of the filters do not work
- on some there are no options
- statuses do not match assignment statuses

https://github.com/user-attachments/assets/063e6367-386a-43c6-986e-9b39c874c75e